src/main/antora/modules/security/pages/security.adoc

```diff
@@ -232,7 +244,7 @@ public class RestBasicAuthCustomStoreIT extends ITBase {
 }
 ----
 
-Using Arquillian, the test starts the default server (GlassFish 7), and deploys the output of the build process (a .war file) to it. The test runs in the integration test phase, rather than the unit test phase, to make sure this build output is available when it runs. The test then sends a request to the remote GlassFish server using the provided HtmlUnit `webClient`. Note that the `webClient` can be used for any other HTTP requests your test requires.
+The test starts a default server and deploys the output of the build process (a .war file) to it. The test runs in the integration test phase, rather than the unit test phase, to make sure this build output is available when it runs. The test then sends a request to the remote GlassFish server using the provided HtmlUnit `webClient`. Note that the `webClient` can be used for any other HTTP requests your test requires.
```

Comment:
Shouldn't "to the remote GlassFish server" be rewritten as "to the default server"? The word "remote" made me think it's a different/remote instance somewhere on the Internet rather than the local one. To take a step further, why the word "default"? It would also be ok to say "starts a server" and "to the server" or "to the started server"?

Reply:
"Default" was used to indicate or forward hint that all tests can be tested against multiple servers (or eventually will be, once we get around to adding additional profiles)

Reply:
"Remote" was used to establish that the test runs client side and the server runs remotely (needs to be accessed via HTTP). This to take away some of the fear that the Arquillian based test is shipped to the server and executed from there.